go/obscuronode/enclave/state.go

```diff
@@ -134,30 +161,14 @@ func updateState(b *types.Block, s Storage, blockResolver BlockResolver) *blockS
 			foundNewRollup: true,
 		}
 		s.SetBlockState(b.Hash(), &bs)
-		return &bs
+		return &bs, true
 	}
 
 	// re-processing the block that contains the rollup
```

Comment:
Can we add more detail to this comment ?

Reply:
what would you have in mind?